### ISSUE TITLE
in iOS ,it is  always redirecting to yobeactive applications in settings.

### ISSUE THREAD

**Issue by ravneet24** [guyromb/Cordova-open-native-settings#32]
In android it is working fine.I want to open iTunes and Appstore page in settings.As mentioned in the plugin we need to give "store" parameter in open function of this plugin.we are using this in our ionic project with this( [https://ionicframework.com/docs/native/open-native-settings/]).It is always redirecting  to yobeactive applications in settings and not to any specific page.
This is the method we have written for this.

 ```
this.openNativeSettings.open("wifi")
      .then(res => {
        console.log("\n\n res: ", res);
      })
      .catch(err => {
        console.log("\n\n Err: ", err);
      });
```

**Comment by hannibaldinski** [comment]
Noticed the same. Running iOS 11.4.1

**Comment by lfx539** [comment]
Noticed the same.Running iOS 11.3.1

**Comment by allisonmachado** [comment]
Noticed the same. Running on iOS 11.4.1

**Comment by Renathesco** [comment]
Noticed the same. Running on iOS 11.4.1

**Comment by TheVictorHugo** [comment]
Noticed the same, Running on iOS 11.4

**Comment by israelor6** [comment]
Notice the same, Running on iOS 11.4.1

**Comment by LennonSantos** [comment]
Noticed the same. Running iOS 11.4.1

**Comment by guyromb** [comment]
@braulio-evangelisti maybe it's better than nothing. Can do a check for iOS version and route everything to this one. As you said, better than nothing. Can you create a pull request for that?

**Comment by TemRhythm** [comment]
Still opening my general settings...

**Comment by braulio-evangelisti** [comment]
Yes @TemRhythm, this change is not supposed to be a fix but a switch back from open App > Settings to open General settings for iOS 11.3 and 11.4. 
In my case, I want to open Location settings but the way it was always opened my apps settings so the user needed to go two steps back and enter locations settings and enable it. 
This way been redirected to general settings it saves the user one step. 
It's not what I was looking for but as @guyromb said, maybe is better than nothing until someone figures out a solution.

**Comment by TemRhythm** [comment]
@guyromb still not working https://youtu.be/KAAym_5KAFY 😢

**Comment by pserna95** [comment]
Any updates on this?

**Comment by guyromb** [comment]
Please read the expected behaviour. There is no fix due to fault of Apple.
At least it allows to open the general settings. 

There will be no fix unless Apple changes it again, or anyone comes with a work around / idea how to implement it natively.
